vermin/processor.py

```diff
@@ -22,12 +23,13 @@ def process_individual(args):
   mins = None
   text = ""
   novermin = set()
+  bps = set()
   with open(path, mode="rb") as fp:
     try:
       parser = Parser(fp.read(), path)
       (node, mins, novermin) = parser.detect()
     except KeyboardInterrupt:
-      return (path, mins, text)
+      return (path, mins, text, bps)
 
     # When input isn't python code, ignore it.
     except ValueError:
```

Comment:
There are two problematic return statements: `return (None, None, None)`.

They both return a 3-tuple, since this functions signature has been changed everything expects it to return 4-tuples.

Reply:
I was getting this stacktrace when using this branch:
```
Traceback (most recent call last):
  File "/home/shawn/Documents/vermin_backports/bin/vermin", line 11, in <module>
    load_entry_point('vermin', 'console_scripts', 'vermin')()
  File "/home/shawn/Documents/vermin_backports/vermin/vermin/main.py", line 44, in main
    (mins, incomp, unique_versions, backports) = processor.process(paths, processes)
  File "/home/shawn/Documents/vermin_backports/vermin/vermin/processor.py", line 83, in process
    pool.imap(process_individual, ((path, config) for path in paths)):
ValueError: need more than 3 values to unpack
```

I made the changes locally and no longer got this error.

Reply:
Thanks for the feedback! I missed those two around:
```py
    # When input isn't python code, ignore it.
    except ValueError:
      # source code string cannot contain null bytes
      return (None, None, None)  # << here
    except TypeError:
      # compile() expected string without null bytes
      return (None, None, None)  # << here
```
Pushing a fix now.

Reply:
Please try again now, but I can't see any other 3-values now. You can resolve the issue and then I'll merge the PR. :)

Reply:
Just gave it a try, it works great :+1: